Makefile

```diff
@@ -32,11 +32,17 @@ ifeq ($(PYTHON),)
 endif
 PYTHON_BIN := $(shell which $(PYTHON))
 PYTHON_VER := $(word 2,$(shell $(PYTHON) -V 2>&1))
+GO_MIN_VER := 1.5
+GO_VER := $(subst go,, $(word 3,$(shell go version 2>&1)) )
 
 ifeq ($(filter 2.7.%,$(PYTHON_VER)),)
   $(error unsupported Python version $(PYTHON_VER), Grumpy only supports 2.7.x. To use a different python binary such as python2, run: 'make PYTHON=python2 ...')
 endif
 
+ifneq "$(GO_MIN_VER)" "$(word 1, $(sort $(GO_MIN_VER) $(GO_VER)))"
```

Comment:
This is clever! It took me a minute to figure it out :)

I think it'd be better if the comparison was more explicit:

```
GO_VER := $(shell go version | sed 's/.*go\([0-9][0-9]*\).\([0-9][0-9]*\).*/\1 \2/')
GO_VER_MAJOR := $(word 1,$(GO_VER))
GO_VER_MINOR := $(word 2,$(GO_VER))
```

And then somehow use `test $(GO_VER_MAJOR) -lt 1 || test $(GO_VER_MINOR) -lt 6` to do the comparison.